docs: start a developer's doc with a tip related to building sambacc dev branches (#156)
samba-in-kubernetes/samba-container: docs/development.md

# Development Guide

## Building samba containers with unreleased sambacc code

> When developing sambacc there are a suite of unit tests but that often not

Reviewer: ```suggestion
Changes to  `sambacc` are validated by  a suite of unit tests but that is often not
```

Author: done

> enough. One may want to try out a unreleased branch of sambacc code as

Reviewer: ```suggestion
enough to ensure  a minimum level of quality for use in samba container images. One may nonetheless want to try out an unreleased branch of sambacc code as
```

Author: adapted

part of a samba container image. Two methods of doing this are:
* Build sambacc RPMs and put them in a yum/dnf repo
* Customize the Containerfile Use a sambacc build stage

Both methods make use of the sambacc build image, part of the sambacc repo

> and available at quay.io: quay.io/samba.org/sambacc

Author: loosely adapted
